src/content/pages/knowledgebase/linux/ivpn-on-qubes-os.md

```diff
@@ -0,0 +1,153 @@
+---
+title: IVPN on Qubes OS - IVPN Help
+h1: IVPN on Qubes OS
+url: /knowledgebase/linux/ivpn-on-qubes-os/
+sections:
+    - linux
+sectionTitle: Linux
+layout: help-details
+weight: 8
+---
+# IVPN on Qubes OS
+
+<div markdown="1" class="notice notice--warning">
+This guide was produced using Qubes OS release 4.1.1 (R4.1) and fedora-36 as VM template
+</div>
+
+### Introduction
+
+Qubes OS uses a ProxyVM as an intermediary between other VMs in the system and the outside world. The primary function of a ProxyVM is to handle network traffic for other VMs, allowing those VMs to remain isolated from the internet, untrusted and potentially hostile networks.
+
+In Qubes OS, each VM is assigned a specific role, such as a NetVM (for networking) or an AppVM (for running applications). When an AppVM needs to connect to the internet, it sends its network traffic to the designated NetVM.
+
+Alternatively, an AppVM can send its network traffic directly to a ProxyVM for processing. This configuration, known as a "ProxyVM-based NetVM," allows the ProxyVM to enforce various security policies, such as blocking certain types of traffic or limiting access to specific resources. It also allows the use of a Virtual Private Network (VPN) connection, which can provide additional privacy and security for network traffic.
+
+By configuring a ProxyVM to use a VPN connection, all network traffic from the other VMs in the system can be routed through the VPN, ensuring that the traffic is encrypted and anonymized before it leaves the system. This approach can be especially useful when accessing sensitive or confidential information over untrusted networks.
+
+In summary, using a Proxy VM in Qubes OS can provide a flexible and customizable framework for managing network security and isolation, including the ability to use a VPN connection for added privacy and security.
+
+This guide describes the configuration for the following setup:
+
+<div markdown="1" class="notice notice--info">
+App VM/s -> Proxy VM (with IVPN) -> Internet
+</div>
+
+### Creating ProxyVM
+
+1. Navigate to Qube manager (`Main menu` -> `Qubes Tools` -> `Qube Manager`):
+
+2. Create “New qube” with the following parameters: 
+
+    - Name and label: **ivpn-proxy**
+    - Type: **StandaloneVM (fully persistent)**
+    - Template: **fedora-36**
+    - Networking: **default (sys-firewall)**
+    - Advanced tab -> “Provide network access to other qubes“: **enabled**
+
+3. Press `OK`
+
+### Installing IVPN client
+
+1. Start the recently created **ivpn-proxy** cube (`right-click` -> `Start/Resume`)
+
+2. Open **ivpn-proxy** terminal (`Main menu` -> `Service: ivpn-proxy` -> `ivpn-proxy: Terminal`)
+
+3. Install IVPN client (refer to <a href="/apps-linux/#fedora" target="_blank">Fedora setup instructions</a>)
```

Comment:
I think we need to show concrete commands directly in this document.
The link to Fedora's instructions is OK. It describes how to install base or/and base+UI packages.

 But for the current document is important that the user will install both package's (base+UI):
```
# Add the IVPN repository
sudo dnf config-manager --add-repo https://repo.ivpn.net/stable/fedora/generic/ivpn.repo

# Install IVPN software (CLI and UI)
sudo dnf install ivpn-ui
```